charts/prompt-keda/templates/scaled-job.yaml

```diff
@@ -124,6 +124,9 @@ spec:
                 value: /app/s3/credentials
               {{- end }}
               {{- if .Values.s3.aws_profile }}
+                {{- if not $.Values.s3.cred_file_auth }}
+                  {{- fail "When s3.aws_profile is set, s3.cred_file_auth must be true." }}
+                {{- end }}
```

Comment:
This seems like this would make it harder to switch between file- and env-based authentication. Does any harm result from setting `AWS_PROFILE` when `AWS_SHARED_CREDENTIALS_FILE` is not defined?